dipy/denoise/adaptive_soft_matching.py

```diff
@@ -3,7 +3,7 @@
 from dipy.core import wavelet
 
 
-def adaptive_soft_matching(ima, fimau, fimao, sigma):
+def adaptive_soft_matching(ima, fimau, fimao, sigma):6
```

Comment:
typo: 6 ?

Reply:
Hello @skoudoro,
I'm sorry about the typo; I have edited it now.

Also, There were many checks that were failing. Is it an issue with my PR?